<launch>
    <node pkg="suction_controller" type="suction_controller_node" name="suction_controller_node" output="screen"/>
</launch>